Classes/GlobalStateExplorers/FLEXFileBrowserTableViewController.m

```diff
@@ -217,9 +217,14 @@ - (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath
                 drillInViewController = [[FLEXWebViewController alloc] initWithText:prettyString];
             } else if ([FLEXWebViewController supportsPathExtension:pathExtension]) {
                 drillInViewController = [[FLEXWebViewController alloc] initWithURL:[NSURL fileURLWithPath:fullPath]];
-            } else if ([[subpath pathExtension] isEqualToString:@"db"]) {
+            } else if ([@[@"db", @"sqlite", @"sqlite3"] containsObject:[subpath pathExtension]]) {
```

Comment:
What do you think about moving this to something like `+[FLEXTableListViewController supportsPathExtension:]`? It would match the pattern used by the web view controller, and it would be nice to keep the supported path extension logic close to `-[FLEXTableListViewController databaseManagerForFileAtPath:]` which is similar.